src/preferences/dialog/dlgprefcontrols.cpp

```diff
@@ -70,12 +70,23 @@ DlgPrefControls::DlgPrefControls(QWidget * parent, MixxxMainWindow * mixxx,
     if (!m_pConfig->exists(ConfigKey("[Controls]","PositionDisplay")))
         m_pConfig->set(ConfigKey("[Controls]","PositionDisplay"),ConfigValue(0));
```

Comment:
Here we should also use the TrackTime::DisplayMode class.
This value does not match the value set in slotResetToDefaults()